src/webxdc/webxdc_tests.rs

```diff
@@ -329,69 +328,6 @@ async fn test_webxdc_contact_request() -> Result<()> {
     Ok(())
 }
 
-#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
-async fn test_webxdc_update_for_not_downloaded_instance() -> Result<()> {
```

Comment:
This can be modified to employ message reordering instead of a partially downloaded instance

Reply:
So I remove this test now again and we add this as todo question in the pre-message tracking issue?